BiometricLocker/BiometricLocker.swift

```diff
@@ -15,30 +15,31 @@ public protocol AuthenticationDelegate: class {
     func didFailAuthentication(error: LAError)
 }
 
-open class BiometricLocker {
+final public class BiometricLocker {
     public enum Key: String {
-        case applicationDidEnterBackgroundDate
+        case applicationDidEnterBackgroundDate = "com.bakkenbaeck.BiometricLocker.applicationDidEnterBackgroundDate"
     }
 
-    open weak var delegate: AuthenticationDelegate?
+    public weak var delegate: AuthenticationDelegate?
 
-    open var policy: LAPolicy = .deviceOwnerAuthenticationWithBiometrics
+    public var policy: LAPolicy = .deviceOwnerAuthenticationWithBiometrics
 
-    open var localizedReason: String = ""
+    public var localizedReason: String = ""
 
-    open var authenticationContext: LAContext {
+    private var authenticationContext: LAContext {
         // Prevents re-using an `LAContext`, once it can no longer evaluate our policy.
         // Just reusing the `LAContext` can cause it to call the success completion block
         // without the user having to enter their biometric ID.
         if !self._authenticationContext.canEvaluatePolicy(self.policy, error: nil) {
-            self._authenticationContext = LAContext()
+            self._authenticationContext = self.newContext()
         }
 
         return self._authenticationContext
     }
 
-    // We do this to prevent a bug where quickly re-using a LAcontext
-    private var _authenticationContext = LAContext()
+    private lazy var _authenticationContext: LAContext = {
```

Comment:
why make this `lazy`?

Reply:
Should probably document/implement this better, and make it a simple var.